java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -4900,4 +4900,28 @@ public void sintercard(BaseClient client) {
                 assertThrows(ExecutionException.class, () -> client.sintercard(badArr).get());
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
+
+    @Test
+    @SneakyThrows
+    public void copy() {
+        assumeTrue(REDIS_VERSION.isGreaterThanOrEqualTo("6.2.0"), "This feature added in redis 6.2.0");
+        // setup
+        String source = "{key}-1" + UUID.randomUUID();
+        String destination = "{key}-2" + UUID.randomUUID();
+
+        // neither key exists, returns 0
```

Comment:
Update comments in tests
```suggestion
        // neither key exists, returns false
```